content/source/docs/enterprise-beta/private/pre-install.html.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Pre-Installation Requirements"
+sidebar_current: "docs-enterprise2-private-preinstall"
+---
+
+# Private Terraform Enterprise Pre-Installation Requirements
+
+Prior to attempting the installation, please read over ALL the documentation. If you have any questions about anything please contact your Sales Engineer or Technical Account Manager. 
+
+You will need to provide the AWS account number\(s\) and region\(s\) you will be deplying into to your SE or TAM at least 1 business day ahead of attempting an installation so that the AMI can be shared with you. In addition you will need access and details for your VCS Provider. Please note the following VCS Providers are supported in Private Terraform:
+
+- GitHub
+- GitLab
+- GitHub Enterprise
+- Bitbucket
+
+*Please note, currently only Bitbucket is supported for the Module Registry. Git support will be added in the near future.* 
```

Comment:
This is ambiguous because the Bitbucket Server support is driven by Git. Should say something like "Support for other VCS providers is coming in a future update." 

But also, are we removing offhand references to the private module registry like this? (@skierkowski)